packages/plugin-coverage/mocks/single-record-lcov.info

```diff
@@ -0,0 +1,33 @@
+TN:
```

Comment:
same as above, can this be shorter?

Reply:
The reason why I created a file with 10 lines is because:
- The coverage is easily divided into %.
- In this test at least 2 entities of each should appear to test the result calculation inside this test. 2 functions do not usually fit into 5 lines so I made it 10.

Usual lcov files describe files of 100+ lines so having one only summarise 10 lines is a minimum in my opinion.

Reply:
Thanks for your elaboration! So it IS small ;)